selfdrive/car/toyota/interface.py

```diff
@@ -234,7 +246,11 @@ def update(self, c):
 
     # gas pedal
     ret.gas = self.CS.car_gas
-    ret.gasPressed = self.CS.pedal_gas > 0
+    if self.CP.enableGasInterceptor:
+    # use interceptor values to disengage on pedal press
+      ret.gasPressed = self.CS.pedal_gas > 15
```

Comment:
what was 15 based on? Noise tolerance?

Reply:
Yes. But the leeway is small enough that even a tap on the gas should still disengage